pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -35,6 +36,18 @@ func (r *RegistryFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&r.Anon, "registry-anon", false, "Set anonymous auth ($IMGPKG_ANON)")
 
 	cmd.Flags().DurationVar(&r.ResponseHeaderTimeout, "registry-response-header-timeout", 30*time.Second, "Maximum time to allow a request to wait for a server's response headers from the registry (ms|s|m|h)")
+
+	cmd.Flags().SetNormalizeFunc(func(f *pflag.FlagSet, name string) pflag.NormalizedName {
+		//TODO: manually test this alias is applying to the correct cmd...
+		switch name {
+		case "azure-container-registry-config":
+			name = "registry-azure-container-registry-config"
+			break
+		}
+		return pflag.NormalizedName(name)
+	})
+
+	cmd.Flags().String("registry-azure-container-registry-config", "", "Path to the file containing Azure container registry configuration information.")
```

Comment:
wdyt? `registry-azure-container-registry-config` -> `registry-azure-cr-config`

i would also add `($IMGPKG_REGISTRY_AZURE_CR_CONFIG)` to the description (similar to other env var setters)

Reply:
done.